data/ddl.sql

```diff
@@ -75,6 +75,7 @@ create table event_sponsors (
 create table user_groups (
     user_id uuid references users(id),
     group_id uuid references groups(id),
+    banned boolean default false,
```

Comment:
It would be useful to convert this into a date time field, so we can keep track of how many time an user has been banned. In the future we could also implement auto unbans (or temporal bans). 
```suggestion
    banned_at timestamptz,
```

Reply:
I totally agree with this change. @dmmulroy @vaibhavsingh97 What do you guys think?

Reply:
> we can keep track of how many time an user has been banned.

`banned_at` field will be updated many times, so how can we track if the user is banned 1 time or 2 times? We will only get the last banned date if we are using `timestampz`

> In the future we could also implement auto unbans (or temporal bans).

So probably we will need to have a separate table for this thing as then we have too many things to consider. Also, maintaining another table will record logs like the reason for banning, no. of times a user gets banned, also the state of ban i.e temp/permanent.